tests/e2e/framework/kubernetes.go

```diff
@@ -1025,6 +1031,30 @@ EOF`, k.KubeConfig, dummyValidationRule)
 	return nil
 }
 
+func (k *KubeInfo) waitForIstioOperator() error {
+
+	get := fmt.Sprintf(`cat << EOF | kubectl --kubeconfig=%s get icp example-istiocontrolplane -n istio-operator
+-o yaml
+EOF`, k.KubeConfig)
+	timeout := time.Now().Add(istioOperatorTimeout)
+	for {
+		if time.Now().After(timeout) {
+			return errors.New("timeout waiting for istio operator to deploy Istio")
+		}
+
+		out, err := util.ShellSilent(get)
+		if err == nil && strings.Contains(out, "Status") {
```

Comment:
shouldn't we be checking for something more explicit like Status=success?

Reply:
The status filed is a struct and will require us to marshal/unmarshal to whole yaml to get the real status. So just make sure we contain Status here.